files/initramfs-tools/union-mount.j2

```diff
@@ -39,10 +39,49 @@ set_tmpfs_log_partition_size()
   [ $maxsize -le $varlogsize ] && varlogsize=$maxsize
 }
 
+whitelist_rw_folder()
+{
+  image_dir=$1
+  whitelist_file=${rootmnt}/host/$image_dir/whitelist
+ 
+  # Return if the whitelist file does not exist
+  if ! test -f "${whitelist_file}"; then
+    return
+  fi
+ 
+  # Return if the secure_boot_enable option is not set
+  if cat /proc/cmdline | grep -v -q "secure_boot_enable=true"; then
```

Comment:
>secure_boot_enable=true [](start = 37, length = 23)

The convention is 
`secure_boot_enable=[0|1]`
or
`secure_boot_enable=[n|y]`

Reply:
Fixed.